CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Changed
+- **BREAKING**: Remove support for HOTP/TOTP secrets - Please use FIDO keys or a dedicated app
```

Comment:
Should we recommend Aegis and andOTP here?

Reply:
Sure thing